modules/cnf-image-based-upgrade.adoc

```diff
@@ -12,16 +12,18 @@ After generating the seed image on the seed cluster, you can move through the st
 * `Upgrade`
 * `Rollback` (Optional)
 
-image::../images/696_OpenShift_Lifecycle_Agent_0624_0.png[Stages of the image-based upgrade]
+.Stages of the image-based upgrade
+image::696_OpenShift_Lifecycle_Agent_0624_0.png[]
```

Comment:
Even though this has a caption, I think it's still best practice to add some sort of alt text in here, even if it's the same text as the caption:
```suggestion
image::696_OpenShift_Lifecycle_Agent_0624_0.png[Stages of the image-based upgrade]
```